demo/opae-nlb-demo/Dockerfile

```diff
@@ -43,13 +43,15 @@ RUN mkdir /install_root \
 
 # Minimal result image
 FROM scratch as final
+ENV PATH="/usr/local/:${PATH}"
+ENV LD_LIBRARY_PATH="/usr/local/:${LD_LIBRARY_PATH}"
 COPY --from=builder /install_root /
 
 # OPAE
 # nlb0 and nlb3 examples
-COPY --from=builder /usr/src/opae/opae-sdk-*/build/bin/nlb* /usr/bin/
+COPY --from=builder /usr/src/opae/opae-sdk-*/build/bin/nlb* /usr/local/
```

Comment:
Should we copy nlb binaries to /usr/local/bin?

Reply:
New commit addresses you comment. nlb0 and nlb3 examples were moved moved from /usr/local/ to /usr/local/bin/